### PR TITLE
[wix-style-react] - <Input /> - improved chrome autofill styles

### PR DESCRIPTION
**Pull request by egidijusja**
### 🔦 Summary

...

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6586

File: packages/wix-style-react/src/Input/Input.st.css [hunk 1]
```diff
@@ -60,6 +60,17 @@
   overflow-block: scroll;
 }
 
+.input:-webkit-autofill,
+.input:-webkit-autofill:hover, 
+.input:-webkit-autofill:focus {
+  -webkit-box-shadow: 0 0 0px 1000px white inset;
```
Comment (mykas) [inline packages/wix-style-react/src/Input/Input.st.css:66]: I believe we have a color variable for white color too: D80

**Review by mykas — APPROVED** [review]
Nice!

**Comment by mykas** [comment]
lets merge this!

**Pull request merged**